engine/graphics/src/vulkan/graphics_vulkan_device.cpp

```diff
@@ -164,18 +161,15 @@ namespace dmGraphics
 
             vkEnumerateDeviceExtensionProperties(vk_device, 0, &vk_device_extension_count, 0);
 
-            // Note: If this is triggered, we need a higher limit for the max extension count
-            //       or simply allocate the exact memory dynamically to hold the extensions data..
-            assert(vk_device_extension_count < vk_max_extension_count);
```

Comment:
My GTX card seems to have more than 128 extensions, so I just made it dynamic instead..